source/fundamentals/time-series.txt

```diff
@@ -20,8 +20,9 @@ Node.js driver.
 Create a Time Series Collection
 -------------------------------
 
-We recommend that you create a time series collection using the MongoDB Shell
-({+mongosh+}). For detailed instructions on creating a time series collection
+We recommend that you create a time series collection using the MongoDB Shell. 
+See the :mdb-shell:`MongoDB Shell documentation<>` to learn more.
```

Comment:
Issue:

The sentence introducing the link doesn't provide the purpose of the cross-reference.
From the [style guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear):

> Link text should begin with the purpose of the cross-reference.

Suggestion:

Explain what the reader can learn more about. E.g.

"Learn more about how to install and run the MongoDB Shell in the :mdb-shell:\`MongoDB Shell documentation <>\`."

Reply:
To fix the branching I'm going to close this PR and open a new one, but will apply all the suggestions you mentioned.